content/docs/guides/nextjs.md

```diff
@@ -116,7 +116,7 @@ export default async function Page() {
 
 ### Pages Router
 
```

Comment:
Depending on how often your data changes, you can use either of these methods for getting your data:
- `getServerSideProps` fetches data at runtime so that content is always fresh.
- `getStaticProps` pre-renders pages at build time for data that is static or changes infrequently.

#### `getServerSideProps`

Reply:
Makes sense to me!